src/server/scripts/OutdoorPvP/OutdoorPvPNA.cpp

```diff
@@ -664,18 +856,55 @@ void OPvPCapturePointNA::UpdateWyvernRoostWorldState(uint32 roost)
 class OutdoorPvP_nagrand : public OutdoorPvPScript
 {
 public:
-    OutdoorPvP_nagrand()
-        : OutdoorPvPScript("outdoorpvp_na")
-    {
-    }
+    OutdoorPvP_nagrand() : OutdoorPvPScript("outdoorpvp_na") { }
 
     OutdoorPvP* GetOutdoorPvP() const override
     {
         return new OutdoorPvPNA();
     }
 };
 
+struct outdoorpvp_na_halaa_creatures : public ScriptedAI
+{
+    outdoorpvp_na_halaa_creatures(Creature* creature) : ScriptedAI(creature) { }
+
+    void UpdateAI(uint32 diff)
```

Comment:
```suggestion
    void UpdateAI(uint32 diff) override
```

Reply:
need override and diff is not used

Reply:
i saw this a lot of times but dunno why, but after your comment now i get the idea :)